cppapi/server/tango_config.h

```diff
@@ -153,6 +153,7 @@
                 #define HAS_THREAD
                 #define HAS_TYPE_TRAITS
                 #define HAS_VARIADIC_TEMPLATE
+                #define HAS_ATTRIBUTE_SPECIFIERS
```

Comment:
According to the comment on line 142, shouldn't HAS_ATTRIBUTE_SPECIFIERS be defined for GCC >= 4.8 instead of >4.3?

Reply:
@t-b , this is probably the reason why Travis CI tests failed on Debian 7, where the compiler is gcc 4.7.2.

Reply:
@bourtemb Yes you are totally right. Looks like my conflict resolution messed that up again. Fixed.